app/boards/shields/cradio/default.keymap

```diff
@@ -0,0 +1,19 @@
+#include <behaviors.dtsi>
```

Comment:
This is an older file naming, and isn't used any more. The previous `cradio.keymap` you added is the one that will actually be used by ZMK. Remove this one please.